proto/message.py

```diff
@@ -602,6 +602,35 @@ def __setattr__(self, key, value):
         if pb_value is not None:
             self._pb.MergeFrom(self._meta.pb(**{key: pb_value}))
 
+    def copy_from(self, other):
+        """Equivalent for protobuf.Message.CopyFrom
+
+        Args:
+            other: (Union[dict, ~.Message):
+                A dictionary or message to reinitialize the values for this message.
+        """
+        if isinstance(other, type(self)):
+            # Just want the underlying proto.
+            other = Message.pb(other)
+        elif isinstance(other, type(Message.pb(self))):
```

Comment:
Why does it `pass` in this scenario?

Reply:
Because there's no transformation that needs to occur. It's just here to catch the input type and make sure that it doesn't generate the type mismatch exception below.

Reply:
👍🏻